README.md

```diff
@@ -2,48 +2,50 @@
 
 #### Table of Contents
 
-1. [Overview](#overview)
-2. [Module Description - What the module does and why it is useful](#module-description)
-3. [Setup - The basics of getting started with tagmail](#setup)
-4. [Usage - Configuration options and additional functionality](#usage)
-6. [Limitations - OS compatibility, etc.](#limitations)
-7. [Development - Guide for contributing to the module](#development)
+1. [Module Description - What the module does and why it is useful](#module-description)
+2. [Setup - The basics of getting started with tagmail](#setup)
+  * [Requirements](#requirements)
+  * [Beginning with tagmail](#beginning-with-tagmail)
+3. [Usage - Configuration options and additional functionality](#usage)
+4. [Limitations - OS compatibility, etc.](#limitations)
+5. [Development - Guide for contributing to the module](#development)
 
-## Overview
-
-The tagmail module forwards Puppet log messages via email if they include specific tags. It is a replacement for Puppet's built-in tagmail report processor.
 
 ## Module Description
 
-The tagmail module replaces Puppet's built-in tagmail feature, which is broken in the JVM-based PE 3.7 and will be completely removed in PE 4.0. The tagmail module provides the same functionality as the tagmail feature.
-
-Tags let you set context for resources, classes, and defines. For example, you can assign a tag to all resources associated with a particular operating system, location, or other characteristic. The tag is then included in all log messages related to those resources. [Read more about tags.](http://docs.puppetlabs.com/puppet/latest/reference/lang_tags.html)
+The tagmail module forwards Puppet log messages as email if the log messages include specific tags.
```

Comment:
...forwards Puppet log messages as email if the log messages relate to resources that have been assigned specific tags.